code/__DEFINES/is_helpers.dm

```diff
@@ -235,6 +235,8 @@ GLOBAL_LIST_INIT(turfs_openspace, typecacheof(list(
 
 #define isprox(O) (istype(O, /obj/item/assembly/prox_sensor))
 
+#define isvoice(O) (istype(O, /obj/item/assembly/voice))
```

Comment:
this name is bad. please just use istype directly we need to stop making more of these when theyre only used by like 2 things